morpheushelper/cogs/alert_channel.py

```diff
@@ -0,0 +1,137 @@
+import logging
+from typing import Optional
+
+from PyDrocsid.settings import Settings
+from PyDrocsid.translations import translations
+from discord import TextChannel, Embed, Member, VoiceState
+from discord.ext import commands, tasks
+from discord.ext.commands import Cog, Bot, Context
+
+from colours import Colours
+from permissions import Permission
+from util import send_to_changelog
+
+
+async def get_max_hops() -> int:
+    """
+    Retrieves the channel hops per minute in order for a message to appear
+    """
+    return await Settings.get(int, key="alert_channel_warn_channel_hops", default=5)
+
+
+class AlertChannelCog(Cog):
+    def __init__(self, bot: Bot):
+        self.bot = bot
+        self.user_hops: dict[int, int] = {}
+
+    async def on_ready(self):
+        try:
+            self.hop_loop.start()
+        except RuntimeError:
+            self.hop_loop.restart()
+
+    async def get_alert_channel(self) -> Optional[TextChannel]:
+        """
+        Retrieves the alert-channel of the specified guild
+        """
+        alert_channel_id: int = await Settings.get(int, key="alert_channel", default=-1)
```

Comment:
keyword args -> positional args
```suggestion
        alert_channel_id: int = await Settings.get(int, "alert_channel", -1)
```